core/store/migrate/migrations/0100_bootstrap_config.sql

```diff
@@ -0,0 +1,117 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE bootstrap_contract_configs
+(
+    bootstrap_spec_id       INTEGER PRIMARY KEY,
+    config_digest           bytea                    NOT NULL,
+    config_count            bigint                   NOT NULL,
+    signers                 bytea[],
+    transmitters            text[],
+    f                       smallint                 NOT NULL,
+    onchain_config          bytea,
+    offchain_config_version bigint                   NOT NULL,
+    offchain_config         bytea,
+    created_at              timestamp with time zone NOT NULL,
+    updated_at              timestamp with time zone NOT NULL,
+    CONSTRAINT bootstrap_contract_configs_config_digest_check CHECK ((octet_length(config_digest) = 32))
+);
+
+ALTER TABLE ONLY bootstrap_contract_configs
+    ADD CONSTRAINT bootstrap_contract_configs_oracle_spec_fkey
+        FOREIGN KEY (bootstrap_spec_id)
+            REFERENCES bootstrap_specs (id)
+            ON DELETE CASCADE;
+
+-- add missing unique constraint to bootstrap specs
+CREATE UNIQUE INDEX idx_jobs_unique_bootstrap_spec_id ON jobs USING btree (bootstrap_spec_id);
+
+-- migrate existing OCR2 bootstrap jobs to the new bootstrap spec
+-- create helper column
+ALTER TABLE bootstrap_specs
+    ADD COLUMN job_id INTEGER;
+
+-- insert bootstrap specs
+INSERT INTO bootstrap_specs (contract_id, relay, relay_config, monitoring_endpoint, blockchain_timeout,
+                             contract_config_tracker_poll_interval, contract_config_confirmations, created_at,
+                             updated_at, job_id)
+SELECT ocr2.contract_id,
+       ocr2.relay,
+       ocr2.relay_config,
+       ocr2.monitoring_endpoint,
+       ocr2.blockchain_timeout,
+       ocr2.contract_config_tracker_poll_interval,
+       ocr2.contract_config_confirmations,
+       ocr2.created_at,
+       ocr2.updated_at,
+       jobs.id
+from jobs
+         INNER JOIN offchainreporting2_oracle_specs as ocr2 ON jobs.offchainreporting2_oracle_spec_id = ocr2.id
+WHERE ocr2.is_bootstrap_peer is true;
```

Comment:
This query seems to not do anything?

Reply:
Thats part of the INSERT INTO above, it selects what data to insert